src/libs/HeaderUtils.ts

```diff
@@ -7,16 +6,12 @@ import * as Session from './actions/Session';
 import * as Localize from './Localize';
 
 type MenuItem = {
-    icon: string | IconAsset;
+    icon: IconAsset;
     text: string;
     onSelected: () => void;
 };
```

Comment:
Remove this type in favor of `ThreeDotsMenuItems`. Rename `ThreeDotsMenuItems` to `ThreeDotsMenuItem`